src/nectar/account.py

```
@@ -613,26 +614,37 @@ def get_reputation(self):
+            if rep_data is not None:
+                # Handle both dict and direct responses
+                if isinstance(rep_data, dict) and "reputation" in rep_data:
+                    return float(rep_data["reputation"])
+                elif isinstance(rep_data, (int, float)):
+                    return float(rep_data)
                 else:
-                    rep = 0
-        else:
+                    # Try to convert to float if it's a string
+                    return float(rep_data)
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)